internal/action/insert_test.go

```diff
@@ -117,17 +113,14 @@ func TestInsert(t *testing.T) { //nolint:paralleltest
 		buf.Reset()
 	})
 
-	t.Run("insert baz via stdin w/ yaml and no input parsing", func(t *testing.T) { //nolint:paralleltest
-		ctx = ctxutil.WithShowParsing(ctx, false)
+	t.Run("insert baz via stdin w/ yaml", func(t *testing.T) { //nolint:paralleltest
 		require.NoError(t, act.cfg.Set("", "core.showsafecontent", "false"))
 		assert.NoError(t, act.insertStdin(ctx, "baz", []byte("foobar\n---\nuser: name\nother: 0123"), false))
 		buf.Reset()
 
 		assert.NoError(t, act.show(ctx, gptest.CliCtx(ctx, t), "baz", false))
-		assert.Equal(t, "foobar\n---\nuser: name\nother: 0123", buf.String())
+		assert.Equal(t, "foobar\n---\nother: 83\nuser: name\n", buf.String())
```

Comment:
This is how YAML works. I think introducing YAML support in the past was a mistake.

I wonder if there is a point keeping support for disabling parsing. And if so: how?
Please remember that this will only apply to content parsed as YAML. AKV should never re-order things.

Reply:
For people who are relying on YAML parsing, being able to access their "raw" data seems like a desirable feature, so being able to disable parsing for YAML should still be supported, no?

Reply:
Yes. I propose we drop the config option `core.parsing` but keep the `-n` flag. WDYT?